test/contextValidation.spec.tsx

```diff
@@ -14,3 +20,40 @@ describe("DAO component renders", () => {
     expect(arc.root.findByType(DAO).props.address).toBe("0x");
   });
 });
+
+describe("Custom loader works", () => {
+  let loader: any;
+  beforeAll(async () => {
+    loader = await create(
+      <Loader
+        render={(props: LoaderProps) => (
+          <div>
+            {props.errors.length > 0
+              ? props.errors.map((error) => error)
+              : "Loading without errors"}
+          </div>
+        )}
+      >
+        <Arc config={arcConfig}>
+          <DAO address="0x">
+            <DAO.Data />
+          </DAO>
+        </Arc>
+      </Loader>
+    );
+  });
+
+  it("Loading component without error", () => {
+    const loaderRoot = loader.root;
+    const errors: Array<null> = [];
+    const loadingDiv = loaderRoot.props.render({ errors }).props.children;
+    expect(loadingDiv).toEqual("Loading without errors");
+  });
+
+  it("Loading component with error", () => {
+    const loaderRoot = loader.root;
+    const errors: Array<string> = ["Cannot connect to blockchain"];
+    const errorDiv = loaderRoot.props.render({ errors }).props.children;
```

Comment:
I don't think this works, since Loader.tsx doesn't have `errors` as a prop it accepts. For this test, we probably want to initialize the DAO with an incorrect address to test the error logging, then also test with a valid address to make sure the loader still renders within errors.

Reply:
Done sir!